near-sdk/tests/store_performance_tests.rs

```diff
@@ -221,6 +221,7 @@ async fn iter() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[allow(clippy::ifs_same_cond)]
```

Comment:
this is a false lint that `since(1.82)` and `since(1.81)` are the same condition